packages/marshaller/src/ddl2/javascriptadapter.ts

```diff
@@ -254,10 +254,10 @@ export class JavaScriptAdapter {
     }
 
     private writeWidget(dataview: DDL2.IView, imports: Imports): string {
-        return `    export const ${dataview.visualization.id} = new ChartPanel()
+        return `    export const ${dataview.visualization.id} = new marshaller.VizChartPanel()
```

Comment:
This seems like a significant change. Why `marshaller_VizChartPanel` instead of `layout_ChartPanel`?

Reply:
It supports minSize and various visibility (like "flyout")